StandardAnalysis/python/customize.py

```diff
@@ -249,7 +248,6 @@ def customize (process,
         process.TriggerWeightProducer.target          =  cms.string  ("")
         process.TriggerWeightProducer.produceMetLeg = cms.bool(False)
         process.TriggerWeightProducer.produceTrackLeg = cms.bool(False)
-        process.TriggerWeightProducer.produceGrandOr = cms.bool(False)
```

Comment:
Github is marking this as removed, but I assume you just changed it to True, right? At least, when I open the file here https://github.com/OSU-CMS/DisappTrks/blob/be74e188e9512d1e1f0eb1929400af5bac113651/StandardAnalysis/python/customize.py, it is what I see. Or you just didn't change it and github is marking things for some reason.

Reply:
yeah not sure why it is marking this line from what I can see it wasn't changed and I never meant to change it

Reply:
Alright. Everything is fine, then